src/pages/home/HeaderView.js

```diff
@@ -111,11 +117,75 @@ function HeaderView(props) {
         if (props.report.stateNum !== CONST.REPORT.STATE_NUM.SUBMITTED && props.report.statusNum !== CONST.REPORT.STATUS.CLOSED && canModifyTask) {
             threeDotMenuItems.push({
                 icon: Expensicons.Trashcan,
+                iconFill: themeColors.icon,
                 text: props.translate('common.cancel'),
                 onSelected: () => Task.cancelTask(props.report.reportID, props.report.reportName, props.report.stateNum, props.report.statusNum),
             });
         }
     }
+
+    if (isChatThread && !isEmptyChat) {
+        if (props.report.notificationPreference === CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN) {
+            threeDotMenuItems.push({
+                icon: Expensicons.ChatBubbles,
+                iconFill: themeColors.icon,
+                text: props.translate('common.joinThread'),
+                onSelected: () => Report.updateNotificationPreference(props.report.reportID, props.report.notificationPreference, CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS, false),
+            });
+        } else if (props.report.notificationPreference.length) {
+            threeDotMenuItems.push({
+                icon: Expensicons.ChatBubbles,
+                iconFill: themeColors.icon,
+                text: props.translate('common.leaveThread'),
+                onSelected: () => Report.leaveRoom(props.report.reportID),
+            });
+        }
+    }
+
+    if (!props.report.isPinned) {
+        threeDotMenuItems.push({
+            icon: Expensicons.Pin,
+            iconFill: themeColors.icon,
```

Comment:
👋 Coming from https://github.com/Expensify/App/issues/28535
This doesn't need an `iconFill`, the color is already applied by default, but passing it here was overriding the behavior when the icon changes color when you hover over it